go/engine/passphrase_recover.go

```diff
@@ -251,6 +251,9 @@ func (e *PassphraseRecover) changePassword(mctx libkb.MetaContext) (err error) {
 	// We either have no server keys or the user is OK with resetting them
 	// Prompt the user for a new passphrase.
 	passphrase, err := e.promptPassphrase(mctx)
+	if err != nil {
+		return err
+	}
```

Comment:
Was this a bug?

Reply:
yes i don't think i the error was intentionally ignored